controller/tokenizer.py

```diff
@@ -19,6 +19,10 @@ def get_doc_bin_in_bytes(
             if not to_be_tokenized:
                 # None / null types can't be tokenized by spacy so dummy string is used
                 to_be_tokenized = ""
+
+            if type(to_be_tokenized) != str:
```

Comment:
out of curiosity, can we check the type using the function `isInstance(to_be_tokenized,str)`?

Reply:
I never used a function called isInstance but seems that there is a similar built-in method provided by python, so most probably yes (btw. Pythonic would be is_instance, even tho the built-in function is called isinstance() what is quite confusing :D)

Reply:
isinstance is the preferred way for most cases

Reply:
changed to use isinstance